packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/components/Controls/Controls.spec.js

```diff
@@ -22,75 +24,84 @@ describe('VariableStarViewer > Component > Controls', function () {
     it('should call props.setYAxisInversion when the flip button is clicked', function () {
       const setYAxisInversionSpy = sinon.spy()
       const wrapper = mount(
-        <Grommet theme={zooTheme}>
-          <Controls setYAxisInversion={setYAxisInversionSpy} />
-        </Grommet>
+        <Controls setYAxisInversion={setYAxisInversionSpy} />,
+        {
+          wrappingComponent: Grommet,
+          wrappingComponentProps: { theme: zooTheme }
+        }
```

Comment:
Out of curiosity, what is the benefit of "wrapping" as an argument to mount rather than explicitly wrapping as before? I know both exist, but I'm not personally aware of the difference.

Reply:
Using these options allows us to wrap a component with a parent component if needed, but guarantees the shallow or fully mounted component for the tests is the intended target. Before the mounted component was `Grommet` and the tests had to always use `wrapper.find(Controls)` to then subsequently test with the intended component. I think there's also some API limitations for the returned components on `find`.